js/scripts.js

```diff
@@ -1,3 +1,293 @@
+    featuredImg: './img/works-img/art-printing.png',
+    technologies: ['html', 'bootstrap', 'ruby'],
+    demoLink: 'https://soniakarungi.github.io/Portfolio/',
+    projectLink: 'https://github.com/SoniaKarungi/Portfolio',
+  },
+  {
+    name: 'Profesional Art Printing Data',
+    descrtiption: "A daily selection of privately personalized reads; no accounts or sign - ups required.has been the industry 's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.",
+    featuredImg: './img/works-img/art-printing.png',
+    technologies: ['html', 'bootstrap', 'ruby'],
+    demoLink: 'https://soniakarungi.github.io/Portfolio/',
+    projectLink: 'https://github.com/SoniaKarungi/Portfolio',
+  },
+  {
+    name: 'Profesional Art Printing Data',
+    descrtiption: "A daily selection of privately personalized reads; no accounts or sign - ups required.has been the industry 's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.",
+    featuredImg: './img/works-img/art-printing.png',
+    technologies: ['html', 'bootstrap', 'ruby'],
+    demoLink: 'https://soniakarungi.github.io/Portfolio/',
+    projectLink: 'https://github.com/SoniaKarungi/Portfolio',
+  },
+  {
+    name: 'Profesional Art Printing Data',
+    descrtiption: "A daily selection of privately personalized reads; no accounts or sign - ups required.has been the industry 's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.",
+    featuredImg: './img/works-img/art-printing.png',
+    technologies: ['html', 'bootstrap', 'ruby'],
+    demoLink: 'https://soniakarungi.github.io/Portfolio/',
+    projectLink: 'https://github.com/SoniaKarungi/Portfolio',
+  },
+  ];
+
+  // First Project divs
+  const firstGridItem = document.createElement('div');
+  const firstImgDiv = document.createElement('div');
+  const firstFeaturedImg = document.createElement('img');
+  const firstProjectDiv = document.createElement('div');
+  const firstHeading = document.createElement('h2');
+  const firstProjectDesc = document.createElement('p');
+  const techSection = document.createElement('ul');
+  const seeProjectButton = document.createElement('ul');
+  const seeProjectLi = document.createElement('li');
+  const seeProjectLink = document.createElement('a');
+  const projectsDiv = document.getElementById('projects');
+
+  // Project details  Modal
+  const projectModal = document.getElementById('project-details');
+
+  /**
+     * grid-item divs
+     */
+  firstGridItem.classList.add('grid-item', 'grid-item1');
+
+  /**
+     * First Project
+     */
+  firstImgDiv.classList.add('first-post-img');
+  firstFeaturedImg.classList.add('placeholder-img');
+  firstHeading.classList.add('post-title');
+  firstProjectDesc.classList.add('post-text');
+  techSection.classList.add('buttons', 'first-buttons');
```

Comment:
- [ ] Kindly align the buttons to the bottom of the pop-up so your design matches the Figma design.
<img width="568" alt="image" src="https://user-images.githubusercontent.com/77190203/161866700-68e553bf-09dc-4e95-8bf4-1c80266f7b4e.png">

Figma design:
<img width="410" alt="image" src="https://user-images.githubusercontent.com/77190203/161866745-4f83af57-e3a5-4b17-b381-8c3350a86025.png">